packages/editor/src/components/block-list/block.js

```diff
@@ -679,7 +679,7 @@ const applyWithSelect = withSelect(
 		const { hasFixedToolbar, focusMode } = getEditorSettings();
 		const templateLock = getTemplateLock( rootClientId );
 		const isParentOfSelectedBlock = hasSelectedInnerBlock( clientId, true );
-		const { name, attributes, isValid } = block;
+		const { name, attributes, isValid } = block || {};
```

Comment:
We should probably leave an inline comment to the effect that this is a very temporary bug fix, where we expect the root issue to be the running of `mapSelectToProps`, but that the component itself would never have a chance to render.